### ISSUE TITLE
[Bug] Cannot view container logs on velaux 

### ISSUE THREAD

**Issue by hnd4r7** [kubevela/kubevela#5264]
**Describe the bug**

I cannot view logs of a test cluster pod container on velaux. It reports "failed to get stream logs pods "xxx" not found". It works fine on the default local cluster.  Commandline "vela ql -q" works fine too.  Seems like there is a bug related to the multicluster log streaming request to the target cluster. 
![image](https://user-images.githubusercontent.com/10682438/210495871-46328d14-25ec-44f0-9110-65947cc14467.png)
![image](https://user-images.githubusercontent.com/10682438/210499830-1868462a-413e-4038-9824-29abb033cb11.png)
![image](https://user-images.githubusercontent.com/10682438/210494993-e08760b1-8544-4b96-a634-727faade711d.png)

```
➜  ~ vela ql -q "collect-logs{cluster=tg-k8s-test, namespace=santa-eaprinter-test, pod=test-nginx-b9d8f45b5-ll5ln, container=test-nginx, previous=false, tailLines=5}"

{
  "logs": "MjAyMi8xMi8zMCAwNDoxMjoyMCBbbm90aWNlXSAxIzE6IHN0YXJ0IHdvcmtlciBwcm9jZXNzIDQxCjIwMjIvMTIvMzAgMDQ6MTI6MjAgW25vdGljZV0gMSMxOiBzdGFydCB3b3JrZXIgcHJvY2VzcyA0MgoyMDIyLzEyLzMwIDA0OjEyOjIwIFtub3RpY2VdIDEjMTogc3RhcnQgd29ya2VyIHByb2Nlc3MgNDMKMjAyMi8xMi8zMCAwNDoxMjoyMCBbbm90aWNlXSAxIzE6IHN0YXJ0IHdvcmtlciBwcm9jZXNzIDQ0CjIwMjIvMTIvMzAgMDQ6MTI6MjAgW25vdGljZV0gMSMxOiBzdGFydCB3b3JrZXIgcHJvY2VzcyA0NQo=",
  "info": {
    "fromDate": "2022-12-30T04:12:00Z",
    "toDate": "2023-01-04T06:10:25Z"
  }
}
```
```
➜  ~ k get configmap -n vela-system collect-logs -o yaml
apiVersion: v1
data:
  template: |
    import (
        "vela/ql"
    )

    collectLogs: ql.#CollectLogsInPod & {
      cluster: parameter.cluster
      namespace: parameter.namespace
      pod: parameter.pod
      options: {
        container: parameter.container
        previous?: parameter.previous
        sinceSeconds?: parameter.sinceSeconds
        sinceTime?: parameter.sinceTime
        timestamps?: parameter.timestamps
        tailLines?: parameter.tailLines
        limitBytes?: parameter.limitBytes
      }
    }
    status: collectLogs.outputs

    parameter: {
      // +usage=Specify the cluster of the pod
      cluster: string
      // +usage=Specify the namespace of the pod
      namespace: string
      // +usage=Specify the name of the pod
      pod: string

      // +usage=Specify the name of the container
      container: string
      // +usage=If true, return previous terminated container logs
      previous: *false | bool
      // +usage=If set, show logs in relative times
      sinceSeconds: *null | int
      // +usage=RFC3339 timestamp, if set, show logs since this time
      sinceTime: *null | string
      // +usage=If true, add timestamp at the beginning of every line
      timestamps: *false | bool
      // +usage=If set, return the number of lines from the end of logs
      tailLines: *null | int
      // +usage=If set, limit the size of returned bytes
      limitBytes: *null | int
    }
kind: ConfigMap
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","data":{"template":"import (\n    \"vela/ql\"\n)\n\ncollectLogs: ql.#CollectLogsInPod \u0026 {\n  cluster: parameter.cluster\n  namespace: parameter.namespace\n  pod: parameter.pod\n  options: {\n    container: parameter.container\n    previous?: parameter.previous\n    sinceSeconds?: parameter.sinceSeconds\n    sinceTime?: parameter.sinceTime\n    timestamps?: parameter.timestamps\n    tailLines?: parameter.tailLines\n    limitBytes?: parameter.limitBytes\n  }\n}\nstatus: collectLogs.outputs\n\nparameter: {\n  // +usage=Specify the cluster of the pod\n  cluster: string\n  // +usage=Specify the namespace of the pod\n  namespace: string\n  // +usage=Specify the name of the pod\n  pod: string\n\n  // +usage=Specify the name of the container\n  container: string\n  // +usage=If true, return previous terminated container logs\n  previous: *false | bool\n  // +usage=If set, show logs in relative times\n  sinceSeconds: *null | int\n  // +usage=RFC3339 timestamp, if set, show logs since this time\n  sinceTime: *null | string\n  // +usage=If true, add timestamp at the beginning of every line\n  timestamps: *false | bool\n  // +usage=If set, return the number of lines from the end of logs\n  tailLines: *null | int\n  // +usage=If set, limit the size of returned bytes\n  limitBytes: *null | int\n}\n"},"kind":"ConfigMap","metadata":{"annotations":{},"name":"collect-logs","namespace":"vela-system"}}
  creationTimestamp: "2022-10-20T09:21:59Z"
  labels:
    oam.dev/render-hash: edbe486f5b5240d1
  name: collect-logs
  namespace: vela-system
  ownerReferences:
  - apiVersion: core.oam.dev/v1beta1
    blockOwnerDeletion: true
    controller: true
    kind: Application
    name: addon-velaux
    uid: 7dfee586-3354-430a-8bd6-c0caffcdb17c
  resourceVersion: "801985384"
  uid: efac9a79-e3cd-4ac7-92bf-8a4cc8bea8e3
```
**To Reproduce**

**Expected behavior**

**Screenshots**

**KubeVela Version**

velaux version 1.6.4
vela core version 1.6.5

```
➜  ~ vela version
CLI Version: v1.6.5
Core Version: 1.6.5
GitRevision: git-18639cc
GolangVersion: go1.19.4
```
```
➜  ~ vela addon status velaux
velaux: enabled (v1.6.4)
==> Installed Clusters
[local]
==> Registry Name
KubeVela
```

**Cluster information**

**Additional context**

**Comment by chivalryq** [comment]
Collecting pod log via velaQL in remote cluster has a bug before, and it is fixed at v1.6.5. This will be fixed after upgrading to VelaUX v1.6.5.
<img width="1218" alt="image" src="https://user-images.githubusercontent.com/47812250/210503370-88270a46-7584-4396-bec1-45fafdfaf941.png">

For now, you can edit the set the apiserver deployment image to v1.6.5 for a quick fix.
```shell
kubectl set image deployment/apiserver -n vela-system apiserver=oamdev/vela-apiserver:v1.6.5
```

**Comment by chivalryq** [comment]
This is caused by duplicated wrapped multicluster RoundTripper for K8s client. See https://github.com/kubevela/pkg/pull/28. And this problem will be resolved in next patch version.
